### DISCUSSION TITLE
Same computer use

### DISCUSSION THREAD

**Discussion by Michael21107** [jullevistrunz/ExternalPoliceComputer#5] [Q&A] [answered]
is EPC optimized for use on the same PC, with a single monitor, running LSPDFR?

**Comment by jullevistrunz** [comment, marked as answer]
No. It was meant to be used on another device like a laptop.
You can use it on the same device though using Steam overlay.

**Reply by Michael21107** [reply]
how is steam overlay compared to another pc?

**Reply by jullevistrunz** [reply]
It's less immersive, I guess. But you can use whatever you prefer

**Reply by Michael21107** [reply]
well u can act like it is the computer in the game lol
thank you
